contracts/src/v0.8/tests/UpkeepCounterStats.sol

```diff
@@ -117,7 +294,23 @@ contract UpkeepCounterStats {
     return (sum, n);
   }
 
-  function getPxDelayLastNPerforms(uint256 p, uint256 n) public view returns (uint256) {
+  function getPxDelayForAllUpkeeps(uint256 p) public view returns (uint256[] memory, uint256[] memory) {
```

Comment:
open idea: this array will quickly go over size. Maybe we explore storing delays by every day